tools/license_header.py

```diff
@@ -100,24 +112,33 @@
                'docs/static_site/src/assets/js/clipboard.js',
                'cmake/upstream/FindCUDAToolkit.cmake',
                'cmake/upstream/select_compute_arch.cmake',
-               'src/operator/numpy/np_einsum_op-inl.h',
 
                # Licensed under 2-Clause BSD in header
                'example/ssd/dataset/pycocotools/coco.py',
 
                # Julia package metadata, generated by Pkg3.jl
                'julia/Project.toml',
-               
+
                # Licensed under Apache 2.0 license
-               'example/image-classification/predict-cpp/image-classification-predict.cc'
+               'example/image-classification/predict-cpp/image-classification-predict.cc',
+
+               # This file
+               'tools/license_header.py',
+
+               # Github template
+               '.github/ISSUE_TEMPLATE/bug_report.md',
+               '.github/ISSUE_TEMPLATE/feature_request.md',
+               '.github/ISSUE_TEMPLATE/flaky_test.md',
+               '.github/PULL_REQUEST_TEMPLATE.md'
```

Comment:
Please add the license header and don't ignore the files.

```
<!--- Licensed to the Apache Software Foundation (ASF) under one -->
<!--- or more contributor license agreements.  See the NOTICE file -->
<!--- distributed with this work for additional information -->
<!--- regarding copyright ownership.  The ASF licenses this file -->
<!--- to you under the Apache License, Version 2.0 (the -->
<!--- "License"); you may not use this file except in compliance -->
<!--- with the License.  You may obtain a copy of the License at -->

<!---   http://www.apache.org/licenses/LICENSE-2.0 -->

<!--- Unless required by applicable law or agreed to in writing, -->
<!--- software distributed under the License is distributed on an -->
<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
<!--- KIND, either express or implied.  See the License for the -->
<!--- specific language governing permissions and limitations -->
<!--- under the License. -->
```

Reply:
Hi @leezu , this piece of license header will invalid the github issue template as I claimed previously, here's an [example](https://github.com/ciyongch/incubator-mxnet/issues/new) by adding [such header](https://github.com/ciyongch/incubator-mxnet/commit/bce3a531a1392623f9808e66f83233278889cb43) to .github templates, you'll see that there's not  any templates in the issue page.  While the normal one is like [this](https://github.com/apache/incubator-mxnet/issues/new/choose). That's why I choose to skip those github template files.
It would be great if you know how to keep the original template functionality after adding the headers?

Reply:
Sorry for the confusion. I'd consider this as a Github bug and you could send them an email. Until then, you're right that we need to exclude the files.